src/components/SquareGridCard.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * A card used with the MUI Grid component that retains a square shape.
+ */
+
+import { Box, Card, CardProps } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { FunctionComponent } from "react";
+
+const StyledGridCard = styled(Card)({
```

Comment:
It looks like this component is a bit larger than the previous song card component.

Reply:
Yeah, the component doesn't have a fixed width, that way the grid items don't overlap on smaller screens. They just take up a percentage of the total width of their container. We can decrease the containe width so that they closer match the mock-ups.